gcb/build/cloudbuild.yaml

```diff
@@ -54,8 +54,8 @@ options:
 substitutions:
   _KUBERNETES_REPO: 'https://github.com/kubernetes/kubernetes.git'
   _KUBERNETES_BRANCH: 'master'
-  _RELEASE_TOOL_REPO: 'https://github.com/kubernetes/release.git'
-  _RELEASE_TOOL_BRANCH: 'master'
+  _TOOL_REPO: 'https://github.com/kubernetes/release.git'
+  _TOOL_BRANCH: 'master'
```

Comment:
@justaugustus we may want to change that to a tag in the future, right?

Reply:
@saschagrunert -- yeah, I'd like this actually eventually be something like `TOOL_REF`, where any valid git ref on the remote should work.